lib/assets/javascripts/src/local-time/helpers/strftime.coffee

```diff
@@ -40,6 +50,23 @@ pad = (num, flag) ->
     else ("0#{num}").slice(-2)
 
 parseTimeZone = (time) ->
+  if longTimeZone = edgeCaseTimeZone(time)
+    knownEdgeCaseTimeZones[longTimeZone]
+  else if shortTimeZone = parseTimeZoneWithIntl(time)
+    shortTimeZone
+  else
+    parseTimeZoneWithHeuristic(time)
+
+edgeCaseTimeZone = (time) ->
+  Object.keys(knownEdgeCaseTimeZones).find (name) ->
+    time.toString().indexOf(name) isnt -1
+
+parseTimeZoneWithIntl = (time) ->
+  return null unless supportsIntlDateFormat
+  result = new Date(time).toLocaleString(undefined, { timeZoneName: "short" }).split(" ").pop()
+  result if result.indexOf("GMT") is -1
```

Comment:
`Date.toLocaleString()` uses `Intl` behind the scenes: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString

It will return `GMT+/-X` when it fails to return a time zone abbreviation.